dockerfiles/ci/alpine_compile_extension/Dockerfile

```diff
@@ -15,10 +15,11 @@ RUN set -eux; \
         libmcrypt-dev \
         libsodium-dev \
         libxml2-dev \
+        gnu-libiconv-dev \
         oniguruma-dev
 
 # Profiling deps
-RUN apk add --no-cache llvm15-libs clang15-dev
+RUN apk add --no-cache llvm16-libs clang16-dev lld
```

Comment:
For the GNU/Linux builds, I need use `llvm-ar`. Do you think I should fork the instructions, or just add `llvm` here so we have the same instructions on both platforms?

Reply:
I'll add `llvm` there as well and rebuild

Reply:
done